[#108532466] Update combined dashboard

mtekel opened the pull request:
[Show number of deployed apps and instances on the dashboard](https://www.pivotaltracker.com/story/show/108532466)
## What

The metric that was requested on this dashboard in this story was already on it. This indicates that the current dashboard is not easy to read/interpret.

Therefore we have made a number of edits to make the dashboard optimized for 1080p.

We have added or changed:
- Router load
- Disk IO state for the runners
- The DEA names were being truncated so we shortened the names
- Add Changed Warden Response time to Warden MS to make sure the values fitted on one line
- Cloud Controller Requests has been moved from App Metrics to Cloud Controller Metrics graph
- Removed Duplicate DEA Metrics `dea_registry_stopping` metric in the App Metrics graph and replaced them with:
  - `dea_registry_crashed`
  - `dea_logging_agent.totalApps`
- Set Y axis to start at zero for the following graphs
  - DEA stats
  - CPU load
  - DEA memory used
  - DEA disk
- Upper limit on Y axis to sane values e.g. 100% or maximum value for a graph
- Title fixes
## Testing

Deploy or apply against existing environment. This dashboard is currently displaying on the main TV so you can compare what you get with what you see. They should be the same apart from the values of the metrics.
## Who

Anyone but @jimconner or @mtekel

mtekel committed:
Update combined dashboard

Update graphs shown on the combined dashboard. Over the past week we
have collected a set of comments from our team members with various
suggestions on how to improve graphs - change scales, change description
of metrics, add or remove some metrics, etc. This is second iteration
of this dashboard and we expect to update it again in the future, as
the usage pattern of our environment changes (more users, different usage
patterns (e.g. CI)) - which is expected to be quite soon.

Added:

* Router load
* Disk IO state for the runners
* The DEA names were being truncated so we shortened the names
* Add Changed Warden Response time to Warden MS to make sure the values fitted on one line
* Cloud Controller Requests has been moved from App Metrics to Cloud Controller Metrics graph
* Removed Duplicate DEA Metrics `dea_registry_stopping` metric in the App Metrics graph and replaced them with:
 * `dea_registry_crashed`
 * `dea_logging_agent.totalApps`
* Set Y axis to start at zero for the following graphs
 * DEA stats
 * CPU load
 * DEA memory used
 * DEA disk
* Upper limit on Y axis to sane values e.g. 100% or maximum value for a graph
* Title fixes

dcarley commented:
> Based on our past experience, we have updated graphs to show metrics
> we think are better suited for observing our trial environment.

Can you explain what experience; why and what is changing in this commit?

mtekel commented:
Our experience of looking on the TV and receiving comments from people on what they (dis)like. We have fixed scales of several graphs, cleaned up some metric description names, added few more metrics etc. You can see the new dash at trial grafana/ `CF combined new` dashboard. I expect we'd go through another iteration of adjusting what we display once we get different usage patterns on that environment (more users, some of them running CI).

dcarley commented:
Can you amend the commit message so that it's recorded in the history?

We need to describe the what and why. Otherwise it's very difficult to objectively review whether this has changed for the better, and there's no reason why someone that hasn't overhead those comments wouldn't unwittingly undo this work.

mtekel commented:
Updated.
